src/api-config/urls.ts

```diff
@@ -0,0 +1,60 @@
+export const url = 
+{
+    footStats: {
+        Calendar: "https://footstatsapiapp.azurewebsites.net/campeonatos/850/calendario",
+    },
+
+    espn: {
+        Calendar: "https://www.espn.com.br/futebol/time/calendario/_/id/2674/santos"
+    },
+
+    apiFootball: {
+/*
+    Vem as informações gerais das partidas, podendo colocar como parametro a season e o time, com isso dentro do JSON podemos capturar isso como importante:
```

Comment:
Remove this whole block of comment for the future, not needed but is a nice to have.